BraveShareTo/ShareToBraveViewController.swift

```diff
@@ -0,0 +1,110 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Social
+import MobileCoreServices
+
+class ShareToBraveViewController: SLComposeServiceViewController {
+    
+    // TODO: Separate scheme for debug builds, so it can be tested without need to uninstall production app.
+    private func urlScheme(for url: String) -> NSURL? {
+        return NSURL(string: "brave://open-url?url=\(url)")
```

Comment:
I _think_ we can start using the Swift's non `NS` classes now. This file uses a bunch and based on the docs, seems we can drop most of the obj-c based stuff. But could be wrong.